src/main/java/core/basesyntax/Factorial.java

```diff
@@ -6,6 +6,11 @@
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+
+        if (number <= 1 || number == 0) {
```

Comment:
do we need this checking: number == 0?

Reply:
done.checking: number == 0 is not needed